Sources/Unbox.swift

```diff
@@ -482,6 +482,25 @@ public class Unboxer {
     
     /// Unbox a required Array containing values of a raw type
     public func unbox<T: UnboxableRawType>(key: String, isKeyPath: Bool = true) -> [T] {
+        if let rawValueArray = UnboxValueResolver<[T]>(self).resolveOptionalValueForKey(key, isKeyPath: isKeyPath) {
+            return rawValueArray
+        }
+        
+        let transformedStringArray = UnboxValueResolver<[String]>(self).resolveOptionalValueForKey(key, isKeyPath: isKeyPath, transform: { stringArray -> [T]? in
+            do {
+                return try stringArray.flatMap {
+                    guard let transformedUnboxedString = T.transformUnboxedString($0) else { throw UnboxValueError.InvalidValue($0, key) }
```

Comment:
Instead of having the overhead of throwables here, wouldn't a ol' classic `for..in` loop with a `return nil` as soon as an invalid value is found do the trick here?